frontend/src/assets/css/main.css

```diff
@@ -2,20 +2,28 @@
 @import url('https://fonts.googleapis.com/css?family=Poppins:100,200,300,400,500,600,700,800,900');
 @import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css');
 
+/* *, *::before, *::after {
+    box-sizing: border-box;
+} */
+
 body {
     margin: 0;
     padding: 0;
     font-family: 'Montserrat', sans-serif;
 }
 
 h1, h2, h3, h4, h5, h6 {
+    font-size: 25pt;
```

Comment:
Please don't set title font-sizes globally. Different titles need different sizes

Reply:
something left from Q, i don't think it actually did something.
Will delete